packages/input-option/src/InputOption/InputOption.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+import { cx } from '@leafygreen-ui/emotion';
+import { useDarkMode } from '@leafygreen-ui/leafygreen-provider';
+import {
+  Polymorphic,
+  PolymorphicAs,
+  usePolymorphic,
+} from '@leafygreen-ui/polymorphic';
+
+import {
+  inputOptionActiveStyles,
+  inputOptionDisabledStyles,
+  inputOptionHoverStyles,
+  inputOptionStyles,
+  inputOptionThemeStyles,
+  inputOptionWedge,
+} from './InputOption.style';
+import { InputOptionProps } from './InputOption.types';
+
+export const InputOption = Polymorphic<InputOptionProps>(
+  (
+    {
+      as = 'li' as PolymorphicAs,
+      children,
+      disabled,
+      focused,
+      active,
+      darkMode: darkModeProp,
+      showWedge = true,
+      isInteractive = true,
+      className,
+      ...rest
+    },
+    ref,
+  ) => {
+    const { Component } = usePolymorphic(as);
```

Comment:
For my own edification.. what's the difference between using the ref passed to the function versus the one returned from the hook?

Reply:
The hook creates a properly typed ref for you to use instead of struggling with typing `useRef`. If you're forwarding a ref then you can ignore the one in the hook